checkov/terraform/module_loading/module_finder.py

```diff
@@ -87,7 +87,7 @@ def _download_module(m):
     distinct_modules = list({m.address: m for m in modules_to_load}.values())
 
     if run_parallel:
-        parallel_runner.run_function(_download_module, distinct_modules)
+        list(parallel_runner.run_function(_download_module, distinct_modules))
```

Comment:
```suggestion
        deque(parallel_runner.run_function(_download_module, distinct_modules), maxlen=0)
```
I think `collections.deque` could be used here, it is faster to evaluate a generator, but I don't think we really need it 😄